packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -82,7 +83,9 @@
   <p class="dummy-paragraph" id="component-api-alert">Here is the API for the component:</p>
 
   <dl class="dummy-component-props" aria-labelledby="component-api-alert">
-    <dt>type <code>enum</code></dt>
+    <dt>
+      type <code>enum</code> <strong class="required">required</strong>
```

Comment:
I used a "strong" tag for the "required" indicator but let me know if you'd prefer to use a "span" or some other tag here.

Reply:
I think `strong` is okay here, always good to have required things clearly marked for screen readers. 👍

Reply:
Thanks!